src/es6/SirenAction.js

```diff
@@ -0,0 +1,161 @@
+import 'd2l-fetch/d2l-fetch.js';
```

Comment:
Hmmm - this is now basically a copy of https://github.com/Brightspace/polymer-siren-behaviors/blob/master/store/siren-action-behavior.js - should they/can they be reconciled somehow? Maybe I should just call into that?

Reply:
probably yes

Reply:
I think I'll not do this for now - the Polymer behaviour generates the now-removed events that `siren-fetch-mixin-let` was also dispatching, which would mean a refactor of that library as well, which I'm not super jazzed about doing. It would also introduce another ES6 -> Polymer dependency, which I'd rather avoid.

In other words, "yes it's a good idea, but not now".

Reply:
Sounds goodish.  Are you going to remove the siren-fetch-mixin-lit soonish or no?

Reply:
Yeah, I was going to do a follow-up PR for it, so that I can do this additive change, then change the consumer over, then do the (technically-breaking) removal of the old stuff. I'm going to be naughty and not bump the major version though, since it's very-new code and is only being used by us (and doing the major version bump everywhere would be bleh).